src/code/sys_cfb.c

```diff
@@ -1,20 +1,128 @@
 #include "global.h"
 #include "system_malloc.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/sys_cfb/func_80178750.s")
+#define D_80780000 gPictoPhotoI8
```

Comment:
What's the point of this define exactly?

Reply:
Probably for my own temporary usage. `D_80780000` had gotten renamed but it doesn't make sense to use the name `gPictoPhotoI8` within the context of this file.

Reply:
Then either `gPictoPhotoI8` should be renamed or `gPictoPhotoI8` should be used. Perhaps `gPictoPhotoI8` could be changed to `D_80780000 ` until we see more. But isn't the picto photo is based on frame buffers though?